.github/workflows/release-publish-documentation.yml

```diff
@@ -0,0 +1,85 @@
+name: Publish downloadable documentation
+
+on:
+  release:
+    types: [released]
+  pull_request:
+    branches: [main, fix-docbuild-zip]
```

Comment:
Maybe we should add a `workflow_dispatch` event here? Also, I think we can also run this on every push and remove the pull-request target since readthedocs builds it anyway.

Reply:
Yes! I think running on every push is overkill, but the `pull_request` trigger was just to test this here. I've replaced it with `workflow_dispatch`. 👍 

Done in 4c6063a.